### PR TITLE
[v16] docs: fix version number

### PR DESCRIPTION
**Pull request by mmcallister**
Due to the backport https://github.com/gravitational/teleport/pull/43396, the version number displayed is incorrect

### CODE REVIEW

**Review by stevenGravy — APPROVED** [review]

File: docs/config.json [hunk 1]
```diff
@@ -2033,6 +2316,16 @@
     }
   },
   "redirects": [
+    {
```
Comment (ptgott) [inline docs/config.json:2319]: Was this PR intended to add these redirects?

File: docs/config.json [hunk 2]
```diff
@@ -31,27 +31,39 @@
             {
               "title": "Teleport Cloud Agents (Linux)",
               "slug": "/upgrading/cloud-linux/",
-              "forScopes": ["cloud"]
+              "forScopes": [
```
Comment (zmb3) [inline docs/config.json:34]: It would be a lot easier to review the change if there weren't unrelated formatting changes. Looks like you might have some sort of auto-formatter configured?

File: docs/config.json (hunk 1 above)
Comment (mmcallister) [reply docs/config.json:2319]: I just grabbed the `config.json` that was in place before my commit, and added the new policy entries.

File: docs/config.json (hunk 2 above)
Comment (mmcallister) [reply docs/config.json:34]: hmm not intentionally, I'm on Visual studio code 1.89.0 (Universal). I'll see if I can disable it.

**Review by zmb3 — APPROVED** [review]
